finality-aleph/src/abft/current/performance/scorer.rs

```diff
@@ -0,0 +1,155 @@
+use current_aleph_bft::{NodeCount, NodeMap, Round};
+
+use crate::{abft::current::performance::Batch, UnverifiedHeader};
+
+pub type Score = Vec<Round>;
```

Comment:
This should be pulled from primitives so that runtime also knows this type.

Reply:
Ah, good point, this was written before the PR with `Score` in the primitives was merged and I forgot to edit in the necessary changes.